### PR TITLE
RC1 for 2018-10-23

### PR DESCRIPTION
**Pull request by MiYanni**
## Description

RC1 for 2018-10-23

## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
+#### AzureRM.Compute
+* Added new sizes to the whitelist of VM sizes for which Accel Net will be turned on when using the simple param set for 'New-AzureRmVm'
+* Added ResourceName argument completer to all cmdlets.
+
+#### AzureRM.Network
+* Update cmdlet Test-AzureRmNetworkWatcherConnectivity, pass the protocol value to backend.
+* Added ResourceName argument completer to all cmdlets.
+
+#### AzureRM.Resources
+* Fix isssue where Get-AzureRMRoleDefinition throws an unintelligible excpetion (when the default profile has no subscription in it and no scope is specified) by adding a meaningful exception in the scenario. Also set the default param set to 'RoleDefinitionNameParameterSet'.
```
Comment (deathly809) [inline ChangeLog.md]: excpetion

File: src/ResourceManager/Compute/Commands.Compute/AzureRM.Compute.psd1 [hunk 2]
```diff
@@ -212,9 +212,8 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Fix Get-AzureRmVM -ResourceGroupName <rg> to return more than 50 results if needed
-* Added an example of the ''SimpleParameterSet'' to New-AzureRmVmss cmdlet help.
-* Fixed a typo in the Azure Disk Encryption progress message'
+        ReleaseNotes = '* Added new sizes to the whitelist of VM sizes for which Accel Net will be turned on when using the simple param set for ''New-AzureRmVm''
```
Comment (deathly809) [inline src/ResourceManager/Compute/Commands.Compute/AzureRM.Compute.psd1]: Accel Net?

File: ChangeLog.md [hunk 3]
```diff
@@ -1,4 +1,23 @@
...
+﻿## 6.11.0 - October 2018
+#### AzureRM.Profile
+* Fix issue with Get-AzureRmSubscription in CloudShell
+* Update common code to use latest version of ClientRuntime
+
+#### AzureRM.Backup
+* Deprecated Azure Backup cmdlets.
+
+#### AzureRM.Compute
+* Added new sizes to the whitelist of VM sizes for which Accel Net will be turned on when using the simple param set for 'New-AzureRmVm'
```
Comment (deathly809) [inline ChangeLog.md]: Accel Net?

File: src/ResourceManager/Compute/Commands.Compute/ChangeLog.md [hunk 4]
```diff
@@ -19,6 +19,8 @@
         - Additional information about change #1
 -->
 ## Current Release
+
+## Version 5.7.1
 * Added new sizes to the whitelist of VM sizes for which Accel Net will be turned on when using the simple param set for `New-AzureRmVm`
```
Comment (deathly809) [inline src/ResourceManager/Compute/Commands.Compute/ChangeLog.md]: Accel Net?

File: src/ResourceManager/Resources/Commands.Resources/AzureRM.Resources.psd1 [hunk 5]
```diff
@@ -171,8 +171,7 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Add missing -Mode parameter to Set-AzureRmPolicyDefinition
-* Fix Get-AzureRmProviderOperation commandlet bug for operations with Origin containing User'
+        ReleaseNotes = '* Fix isssue where Get-AzureRMRoleDefinition throws an unintelligible excpetion (when the default profile has no subscription in it and no scope is specified) by adding a meaningful exception in the scenario. Also set the default param set to ''RoleDefinitionNameParameterSet''.'
```
Comment (deathly809) [inline src/ResourceManager/Resources/Commands.Resources/AzureRM.Resources.psd1]: excpetion

File: src/ResourceManager/Resources/Commands.Resources/ChangeLog.md [hunk 6]
```diff
@@ -18,6 +18,8 @@
         - Additional information about change #1
 -->
 ## Current Release
+
+## Version 6.7.0
 * Fix isssue where Get-AzureRMRoleDefinition throws an unintelligible excpetion (when the default profile has no subscription in it and no scope is specified) by adding a meaningful exception in the scenario. Also set the default param set to `RoleDefinitionNameParameterSet`.
```
Comment (deathly809) [inline src/ResourceManager/Resources/Commands.Resources/ChangeLog.md]: excpetion

File: tools/AzureRM/AzureRM.psd1 [hunk 7]
```diff
@@ -161,52 +161,24 @@ PrivateData = @{
...
+* Deprecated Azure Backup cmdlets.
 
 AzureRM.Compute
-* Fix Get-AzureRmVM -ResourceGroupName <rg> to return more than 50 results if needed
-* Added an example of the ''SimpleParameterSet'' to New-AzureRmVmss cmdlet help.
-* Fixed a typo in the Azure Disk Encryption progress message
-
-AzureRM.DataFactoryV2
-* Updated the ADF .Net SDK version to 2.3.0.
+* Added new sizes to the whitelist of VM sizes for which Accel Net will be turned on when using the simple param set for ''New-AzureRmVm''
```
Comment (deathly809) [inline tools/AzureRM/AzureRM.psd1]: Accel Net.

File: tools/AzureRM/AzureRM.psd1 [hunk 8]
```diff
@@ -161,52 +161,24 @@ PrivateData = @{
...
-* Fixed issue where some backup cmdlets would not recognize the current azure subscription
-
-AzureRM.Storage
-* Support get the Storage resource usage of a specific location, and add warning message for get global Storage resource usage is obsolete.
-    - Get-AzureRmStorageUsage
-
-AzureRM.Websites
-* New Cmdlet Get-AzureRMWebAppContainerContinuousDeploymentUrl - Gets the Container Continuous Deployment Webhook URL
-* New Cmdlets New-AzureRMWebAppContainerPSSession and Enter-WebAppContainerPSSession  - Initiates a PowerShell remote session into a windows container app
+* Fix isssue where Get-AzureRMRoleDefinition throws an unintelligible excpetion (when the default profile has no subscription in it and no scope is specified) by adding a meaningful exception in the scenario. Also set the default param set to ''RoleDefinitionNameParameterSet''.
```
Comment (deathly809) [inline tools/AzureRM/AzureRM.psd1]: excpetion

File: ChangeLog.md (hunk 1 above)
Comment (MiYanni) [reply ChangeLog.md]: Fixed.

File: src/ResourceManager/Compute/Commands.Compute/AzureRM.Compute.psd1 (hunk 2 above)
Comment (MiYanni) [reply src/ResourceManager/Compute/Commands.Compute/AzureRM.Compute.psd1]: Fixed.

File: ChangeLog.md (hunk 3 above)
Comment (MiYanni) [reply ChangeLog.md]: Fixed.

File: src/ResourceManager/Compute/Commands.Compute/ChangeLog.md (hunk 4 above)
Comment (MiYanni) [reply src/ResourceManager/Compute/Commands.Compute/ChangeLog.md]: Fixed.

File: src/ResourceManager/Resources/Commands.Resources/AzureRM.Resources.psd1 (hunk 5 above)
Comment (MiYanni) [reply src/ResourceManager/Resources/Commands.Resources/AzureRM.Resources.psd1]: Fixed.

File: src/ResourceManager/Resources/Commands.Resources/ChangeLog.md (hunk 6 above)
Comment (MiYanni) [reply src/ResourceManager/Resources/Commands.Resources/ChangeLog.md]: Fixed.

File: tools/AzureRM/AzureRM.psd1 (hunk 7 above)
Comment (MiYanni) [reply tools/AzureRM/AzureRM.psd1]: Fixed.

File: tools/AzureRM/AzureRM.psd1 (hunk 8 above)
Comment (MiYanni) [reply tools/AzureRM/AzureRM.psd1]: Fixed.

**Comment by MiYanni** [comment]
powershell-demand: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/powershell-demand/1125/
ps-sign: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/1443/
psnetsign: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/psnetsign/235/

File: src/ResourceManager/AzureBackup/Commands.AzureBackup/AzureRM.Backup.psd1 [hunk 9]
```diff
@@ -1,9 +1,9 @@
 #
-# Module manifest for module 'AzureRM.Backup'
+# Module manifest for module 'PSGet_AzureRM.Backup'
```
Comment (cormacpayne) [inline src/ResourceManager/AzureBackup/Commands.AzureBackup/AzureRM.Backup.psd1]: @MiYanni should we keep this change as-is and report back on the original issue that `Update-ModuleManifest` is causing this issue and we can't fix the header until we move to the mechanism that doesn't use that cmdlet?

File: tools/AzureRM/AzureRM.psd1 [hunk 10]
```diff
@@ -219,7 +191,7 @@ AzureRM.Websites
         # ExternalModuleDependencies = @()
 
     } # End of PSData hashtable
-
+    
  } # End of PrivateData hashtable
 
 # HelpInfo URI of this module
```
Comment (cormacpayne) [inline tools/AzureRM/AzureRM.psd1]: @MiYanni I can't make a comment on the file, but it looks like changes were made to `AzureRM.psm1` that should be reverted. Would you mind checking that removing the beginning `/` character from the entry in the [`.gitignore`](https://github.com/Azure/azure-powershell/blob/preview/.gitignore#L6) file resolves the issue with changes being allowed to be checked in for this file?

File: src/ResourceManager/AzureBackup/Commands.AzureBackup/AzureRM.Backup.psd1 (hunk 9 above)
Comment (MiYanni) [reply src/ResourceManager/AzureBackup/Commands.AzureBackup/AzureRM.Backup.psd1]: Yes. We will no longer be using `Update-ModuleManifest` in the future (for Az). So, the issue will persist for `AzureRM` but will not happen in `Az`. That should just be the result of the issue. We don't need to do anything for AzureRM.

**Pull request merged**